src/pages/EditMonitor.vue

```diff
@@ -96,7 +96,8 @@
                             <div class="my-3">
                                 <label for="parent" class="form-label">{{ $t("Monitor Group") }}</label>
                                 <select v-model="monitor.parent" class="form-select" :disabled="sortedMonitorList.length === 0">
-                                    <option :value="null" selected>{{ $t("None") }}</option>
+                                    <option v-if="sortedMonitorList.length === 0" :value="null" selected>{{ $t("noGroupMonitorMsg") }}</option>
+                                    <option v-else :value="null" selected>{{ $t("None") }}</option>
```

Comment:
I think the other way around this could be a bit cleaner
```suggestion
                                    <option v-if="sortedMonitorList" :value="null" selected>{{ $t("None") }}</option>
                                    <option v-else :value="null" selected>{{ $t("noGroupMonitorMsg") }}</option>
```

Reply:
Empty array would return true which is not the desired condition.

Reply:
WTF, javascript?
I have adjusted the proposed change to reflect this, but would be fine if this comment is ignred ^^